Gordon360/Models/ViewModels/Class.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Gordon360.Models.ViewModels
+{
+    public class OfficeLocationPatchViewModel
+    {
+        public string BuildingDescription { get; set; }
+        public string RoomNumber { get; set; }
+    }
+}
```

Comment:
Why do we call this file Class.cs? Should we call it something like OfficeLocationPatchViewModel.cs?

Reply:
Good catch! Just changed the name